gnosis/safe/safe_signature.py

```diff
@@ -148,10 +148,42 @@ def parse_signature(
             safe_signatures.append(safe_signature)
         return safe_signatures
 
+    @classmethod
+    def export_signatures(cls, safe_signatures: Sequence["SafeSignature"]) -> HexBytes:
```

Comment:
Why this name? In my opinion is too generic, could be better build_eip1271_signature?

Reply:
That's not building a `1271` signature. It's building a valid and accepted signature by the smart contract